### ISSUE TITLE
Soundboard sound on entry

### ISSUE THREAD

**Issue by yourlocalsadmushroom** [Vencord/plugin-requests#197]
### Plugin Description

Obviously cant use sounds that are locked if you dont have nitro but using sounds from that server or the ones discord gives you everywhere.

Basically the nitro feature just without every sound available

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Sqaaakoi** [comment]
Surprisingly enough, this is a rare case where a suggestion related to FakeNitro is actually possible
The client would have to store the sound in the settings and manually trigger the effect after joining

**Comment by yourlocalsadmushroom** [comment]
> Surprisingly enough, this is a rare case where a suggestion related to FakeNitro is actually possible The client would have to store the sound in the settings and manually trigger the effect after joining

yeah! it would just have to be played manually when the user joins
